_cloudstor/03-setup-cloudstor.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Setup AARNET CloudStor
+order: 3
+duration: 5
+---
+
+1. Login to Nectar virtual machine via your prefered termianl software. In this tutorial, we use [Putty](https://putty.org). 
```

Comment:
We shouldn't mention putty here. It would be useful to link this back to a tutorial on using SSH instead where users can get info for their OS platform